server/eclipse-project/src/main/java/us/freeandfair/corla/csv/ColoradoBallotManifestParser.java

```diff
@@ -89,7 +89,7 @@
   /**
    * The county ID to apply to the parsed manifest lines.
    */
-  private final Integer my_county_id;
+  private final Long my_county_id;
```

Comment:
Uniformity in database table "ID" columns being represented as java.lang.Long. At this point I probably _could_ switch the county and dashboard ones to Integer (since they are no longer inheriting from the same parent), but I don't know that I see the point.